util/dependencies.sbt

```diff
@@ -1,10 +1,10 @@
-val catsVersion = "0.9.0"
+val catsVersion = "1.0.0-MF"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2",
-  "joda-time" % "joda-time" % "2.9.9",
-  "org.joda" % "joda-convert" % "1.8.2",
-  "org.typelevel" %% "cats-macros" % catsVersion,
-  "org.typelevel" %% "cats-core" % catsVersion,
-  "org.json4s" %% "json4s-scalap" % "3.5.2"
+  "com.typesafe.scala-logging"  %% "scala-logging"  % "3.7.2",
+  "joda-time"                   %  "joda-time"      % "2.9.9",
```

Comment:
I like this formatting style more. I think it helps too read the dependencies more clearly. 
There is a Github url parameter `w=1` which ignores whitespace changes. I think this should help reading a PR with a lot of whitespace changes.